python/ray/train/torch/config.py

```diff
@@ -46,6 +48,30 @@ def backend_cls(self):
         return _TorchBackend
 
 
+@PublicAPI(stability="alpha")
+@dataclass
+class TorchXLAConfig(BackendConfig):
+    """
+    Configuration for torch XLA setup.
+    See https://pytorch.org/xla/release/1.13/index.html for more info.
+
+    Args:
+        runtime: Runtime (default=XRT) to use for training.
+        device_type: The device type used to differentiate the XLA backend.
```

Comment:
What are the supported values here?

Reply:
1. Added, it's xrt/pjrt.
2. Also, refactor to accelerator_type and included neuron_cores